Gordon360/Documentation/Gordon360.xml

```diff
@@ -1,2576 +1,2713 @@
 <?xml version="1.0"?>
 <doc>
-    <assembly>

```

Comment:
What was this reverted to?  It appears to have changed almost the whole file.  Mostly indentation and whitespace, but I can't really tell.  Anyway, we can't submit this huge change with this PR.